vllm/v1/executor/multiproc_executor.py

```diff
@@ -316,43 +352,25 @@ def signal_handler(signum, frame):
 
             worker.worker_busy_loop()
 
-        except SystemExit:
-            logger.debug("Worker interrupted.")
```

Comment:
are you intentionally not catching this anymore?

Reply:
see other comment about `SystemExit`